go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/aquasecurity/table v1.8.0
 	github.com/aquasecurity/testdocker v0.0.0-20240730042311-4642e94c7fc8
 	github.com/aquasecurity/tml v0.6.1
-	github.com/aquasecurity/trivy-checks v1.1.0
+	github.com/aquasecurity/trivy-checks v1.1.1-0.20241003010744-73c00e1faaea
```

Comment:
@nikpivkin once https://github.com/aquasecurity/trivy-checks/pull/264 is merged, we can release another trivy-checks and bump this up to v1.1.1